lms/templates/discussion/_filter_dropdown.html

```diff
@@ -19,38 +19,42 @@
     <li
         class="forum-nav-browse-menu-item"
         data-discussion-id='${entries[entry]["id"]}'
+        id='${entries[entry]["id"]}'
         data-cohorted="${str(entries[entry]['is_cohorted']).lower()}"
+        role="option"
     >
-        <button type="button" class="forum-nav-browse-title">${entry}</button>
+        <span class="forum-nav-browse-title">${entry}</span>
     </li>
 </%def>
 
 <%def name="render_category(categories, category)">
-    <li class="forum-nav-browse-menu-item">
-        <button type="button" class="forum-nav-browse-title">${category}</button>
+    <li class="forum-nav-browse-menu-item"
+        id='${category}'
+    >
+        <span class="forum-nav-browse-title">${category}</span>
         <ul class="forum-nav-browse-submenu">
             ${HTML(render_dropdown(categories[category]))}
         </ul>
     </li>
 </%def>
 
-<div class="forum-nav-browse-menu-wrapper" style="display: none">
-    <form class="forum-nav-browse-filter">
-        <label>
+<div class="forum-nav-browse-menu-wrapper" style="display: none" aria-label="${_("Discussion topics list")}">
+    <form class="forum-nav-browse-filter" autocomplete="off">
+        <label for="forum-nav-browse-filter-input" aria-label="${_("Filter Topics")}">
             <span class="sr">${_("Filter Topics")}</span>
-            <input type="text" class="forum-nav-browse-filter-input" placeholder="${_("filter topics")}">
+            <input type="text" id="forum-nav-browse-filter-input" role="combobox" aria-expanded="true" aria-owns="discussion_topics_listbox" aria-autocomplete="list" class="forum-nav-browse-filter-input" aria-activedescendant="all_discussions" placeholder="${_("filter topics")}">
             <span class="icon fa fa-filter" aria-hidden="true"></span>
         </label>
     </form>
-    <ul class="forum-nav-browse-menu">
-        <li class="forum-nav-browse-menu-item forum-nav-browse-menu-all">
-            <button type="button" class="forum-nav-browse-title">${_("All Discussions")}</button>
+    <ul class="forum-nav-browse-menu" role="listbox" id="discussion_topics_listbox" aria-expanded="true">
```

Comment:
@alisan617 aria-expanded is not necessary here. It only needs to be on the element that triggers the expansion.